R/outbreak_step.R

```diff
@@ -53,18 +53,18 @@
 #' case_data <- out[[1]]
 #' case_data
 outbreak_step <- function(case_data = NULL, disp_isolated = NULL, disp_community = NULL,
-                          r0isolated = NULL, r0community = NULL,
+                          r0_isolated = NULL, r0_community = NULL,
                           prop_asymptomatic = NULL, incubation_period = NULL,
                           onset_to_isolation = NULL, prop_ascertain = NULL,
-                          k = NULL, quarantine = FALSE, r0asymptomatic = NULL,
+                          k = NULL, quarantine = FALSE, r0_asymptomatic = NULL,
                           disp_asymptomatic = NULL) {
```

Comment:
if approach gets refactored to options-style group of arguments, probably want to group all the isolated vs community vs asymptomatic together in the arguments.

Reply:
...especially since internal invocations, examples, etc all seem to call them grouped (by using explicity argument names).

Reply:
Will do 👍